dependencies/vaticle/repositories.bzl

```diff
@@ -28,7 +28,7 @@ def vaticle_typeql_lang_java():
     git_repository(
         name = "vaticle_typeql_lang_java",
         remote = "https://github.com/vaticle/typeql-lang-java",
-        commit = "c4ecdf860ff05ad8dc053b8746761eba4bb4e49b", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typeql_lang_java
+        commit = "120ce84bbfcdcde3213b3894af258a9814870606", # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typeql_lang_java
```

Comment:
Update to latest commit when https://github.com/vaticle/typeql-lang-java/pull/5 is merged

NOTE: I forgot to use my fork for this one, sorry!